### ISSUE TITLE
Ireland map issues

### ISSUE THREAD

**Issue by adibbz** [amcharts/amcharts4#3536]
`irelandLow` and `irelandHigh` maps appear to be sending back the same data or at least rendering the same map. This is an issue for us because our API is using the 4 Provinces. Not sure if these are meant to be the same - or is there an Ireland Provinces map? 

I'm using the cdn at https://cdn.amcharts.com/lib/4/geodata/json/ but see below for reference: 
irelandLow - [https://www.amcharts.com/lib/4/geodata/json/irelandLow.json](https://www.amcharts.com/lib/4/geodata/json/irelandLow.json)
irelandHigh - [https://www.amcharts.com/lib/4/geodata/json/irelandHigh.json](https://www.amcharts.com/lib/4/geodata/json/irelandHigh.json)

Here's a side by side where I switched out to load both the low and high map. 
![irelandMaps](https://user-images.githubusercontent.com/6921939/119567977-99d92b80-bd72-11eb-9095-e110407c34a0.png)

**Comment by martynasma** [comment]
They are the same map, with different level of precision. We don't have provinces map for Ireland, I'm afraid :(

Is 4-province map more commonly used then the one above?

**Comment by adibbz** [comment]
Ah ok. I'm not sure if it is more common but it would definitely help us a ton. Our geo db doesn't break down Ireland into counties so we aren't able to match our api data with what is available in the amcharts map.

**Comment by martynasma** [comment]
Just to make sure, you need map of Irish provinces?

https://en.wikipedia.org/wiki/Provinces_of_Ireland

**Comment by adibbz** [comment]
Yup that's it. Just the 4 provinces.

**Comment by martynasma** [comment]
Thanks. We'll see if we can add that map, too. Will keep you posted.

**Comment by martynasma** [comment]
We've decided to add Ireland Provinces map. It will be available in the next version of geodata package. I'll ping you once it's out.

**Comment by martynasma** [comment]
Example:
![image](https://user-images.githubusercontent.com/1458520/120145199-cd410d80-c1eb-11eb-91a9-191fe32c4042.png)

**Comment by adibbz** [comment]
Awesome! Thank you 🙏

**Comment by adibbz** [comment]
@martynasma hey marty - has this update been published yet? 

I also had a question with how I'm accessing map data. Currently I load the maps from the cdn like so:
```
this.countrySeries.geodataSource.url = `https://cdn.amcharts.com/lib/4/geodata/json/${map}.json`;
this.countrySeries.geodataSource.load();
```

Will the update eventually make it to that url? Or should I be using this package and referencing it?
[https://www.npmjs.com/package/@amcharts/amcharts4-geodata](https://www.npmjs.com/package/@amcharts/amcharts4-geodata)

**Comment by martynasma** [comment]
These maps were included in geodata package 4.1.21 released just now.
